app/workers/update_monthly_statistics_report.rb

```diff
@@ -4,6 +4,8 @@ class UpdateMonthlyStatisticsReport
   sidekiq_options retry: 0, queue: :low_priority
 
   def perform
+    return unless Time.zone.now.day == 1
```

Comment:
I know this pattern is used elsewhere but might be better to keep this 'run on the 1st of the month only' logic in `clock.rb`. You can do `if: lambda { |t| t.day == 1 }` I think.

Reply:
The downside is that it's not so testable I guess.

Reply:
Yeah I did notice that ... I can move this logic back to clock if that is the accepted pattern